docs/temp-v5issue.html

```diff
@@ -251,7 +251,7 @@ <h2>Reported issue list</h2>
 	<li><del><strong>Fixed - Item 29:</strong> Typo: was "The official website of the Governement of Canada"; should be: "The official website of the Government of Canada", There in not "e" between "n" and "m" in "Government"</del></li>
 	<li><del><strong>Fixed Item 30:</strong> In GCWeb for maintenance - Remove any reference to "focuson" (files and i18n data) because that information wont be re-use.</del></li>
 	<li><del><strong>Won't fix Item 31:</strong> Format the date modified like "{Full month} {Day number}, {4 digit year}". The previous date format would need to be set to the <code>datetime</code> attribute of the current <code>&lt;time&gt;</code> element.</del></li>
-	<li><del><strong>Completed Item 32:</strong> Identify any potential conflict between this redesign and other WET plugin in this theme. Full list of plugins with this themes available : <a href="../demos/index-en.html">GCWeb - WET Plugin demos</a></del></li>
+	<li><del><strong>Completed Item 32:</strong> Identify any potential conflict between this redesign and other WET plugin in this theme. Full list of plugins with this themes available : <a href="https://wet-boew.github.io/themes-dist/GCWeb/demos/index-en.html">GCWeb - WET Plugin demos</a></del></li>
```

Comment:
This URL will need to updated for "gcweb-compiled-demo" repository instead of themes-dist.